src/database/adapters/mongo.ts

```diff
@@ -41,32 +46,36 @@ export const create = (db: string, collection: string): BidDatabaseService => {
     return result.acknowledged;
   };
 
-  const getBidsByTokenIds = async (tokenIds: string[]): Promise<Bid[]> => {
+  const getBidsByTokenIds = async (tokenIds: string[], bidStatus: BidFilterStatus): Promise<CancelableBid[]> => {
```

Comment:
The return type of `Promise<CancelableBid[]>` seems a little misleading if `BidFilterStatus` enum includes a value to return bids that have already been canceled.

Reply:
Im not sure I understand your concern, in this case CancelableBid is providing an interface for bids that can be cancelled.

Normal bids lack a field to indicate cancel status.

Reply:
Gotcha, to elucidate my confusion is that while the name `CancelableBid` communicates the idea of a bid that **can be cancelled**, is a bid that's already been cancelled still a `CancelableBid`?  My initial thought was no. A bid can't be cancelled twice, but the `cancelled` value of `BidFilterStatus` suggests that cancelled bids can be returned by this function.

This is a nit, I do see the distinction you want to create, just the term `CancelableBid` caused some confusion for me.

Reply:
In this context, a cancelled bid is a CancelableBid with the cancelDate filled. 

The reason this interface is used, is because these endpoints must support returning **both** statuses of bids (Cancelled and Active) in the same response.